### DISCUSSION TITLE
Se manter animado e saber qual caminho seguir

### DISCUSSION THREAD

**Discussion by manzoliric** [frontendbr/forum#1769] [Discussão]
Nos ultimos meses (creio que quase 1 ano) estou tendo dificuldades para estudar e se manter animado com os estudos, eu amo o que eu faço, não me sinto desmotivado no trabalho, o problema tem sido em conseguir trabalhar e estudar e o que estudar... alguem já passou por isso?

Creio que estou num ponto que não sei mais o que aprender, o que mais sobre Front-end devo aprender, Javascript, CSS, estou tentando pensar num jeito de nivelar meu conhecimento e o que preciso aprender para melhorar o meu codigo ou o meu conhecimento em Javascript.

**Comment by DyogoViana** [comment]
Se o foco do aprendizado for para futuras vagas de trabalho, eu aconselho procurar ver o que as que vc gosta, estão pedindo.

Mas se o objetivo tiver em aberto, eu aconselho procurar ouvir **podcast** (_ex: Hipsters Ponto Tech, Podcast Faladev, Lambda3 , BrazilJS, Carreira Sem Fronteiras, Like a Boss, Learn to Code With Me)_
ou **Youtube** _(ex: filipe deschamps, Lucas Montano, Programador BR, Fun Fun Function...)._

**Comment by mstuttgart** [comment]
@manzolirch estou no mesmo barco que você a quase dois anos :smile: . 

Eu ainda não consegui identificar precisamente o gatilho desse desânimo, uma vez que sempre fui muito havido por aprender coisas novas, ainda mais sobre TI. Li em um livro chamado ["A Educação da Vontade"](https://www.skoob.com.br/a-educacao-da-vontade-812905ed816965.html) algo que respondeu, em parte, essa minha dúvida. Ele fala sobre o impacto da experiência em nossa vontade de aprender. Ele cita alguns exemplos de profissões e atividades em que, conforme vamos adquirindo mais experiência, vamos perdendo o interesse de aprender coisas novas, uma vez que a experiência que temos na área nos é suficiente para resolver a maioria dos problemas que encontramos no caminho. Percebo que parte do meu "desanimo" é por isso, apesar de como você, ser apaixonado pelo que faço.

Tenho em todos esses meses lido e pesquisado muito sobre estrategias de estudo e métodos de aprendizado. O que tem funcionado pra mim é focar em estudar apenas um assunto, dividi-lo em  tarefas beeem pequenas e que seja algo que eu possa fazer uso prático (no trabalho, por exemplo). Este [roadmap](https://github.com/kamranahmedse/developer-roadmap) tem um guia bem legal do que estudar, eu tendo me pautar por ele (sou dev backend). Quando algum projeto do serviço requer um conhecimento que eu não domino, eu também tento utilizar isso como motivação pra aprender algo novo.  

Outro ponto que tenho percebido nessa jornada de reencontro a minha motivação de estudar é que faz muita diferença conversar com outras pessoas da área. Então, participar de fóruns, projetos de código aberto, ir a eventos e assistir palestras também ajuda a se manter motivado (algo muito importante se você trabalha remoto, que é o meu caso). 

Pra finalizar, um ponto que acho interessante destacar, é o nosso próprio amadurecimento como individuo. Apesar de ser importante sempre nos mantermos atualizados com as novas tecnologias que surgem, vale a pena refletir se nosso gosto pelo objeto de estudo mudou. No meu caso, por exemplo, de uns anos pra cá, comecei a me interessar muito por Filosofia e Teologia, e percebi que isso afetou bastante meu interesse em aprender outras coisas.

Bom, já falei demais :grin: . Também compartilho das mesmas dúvidas que você e gostaria muito de ver a opinião do restante do pessoal do fórum!

**Comment by Fziliotti** [comment]
@manzolirch De vez em quando passo por momentos como esse que descreveu, o que funciona para mim e aconselho você tentar é basear  seus estudos em pequenos projetos. Deixa de lado essas checklists e roadmaps por um momento e foca em desenvolver um projeto que achar interessante, pode ser uma cópia de algum sistema que você conhece  ou pode ser um projetinho para automatizar alguma coisa que você faz muito manualmente. Eu por exemplo recentemente aprendi a criar a um plugin para o chrome e achei bem legal, também fiz um projeto para mostrar os casos do covid-19 no páis, estados e cidades.. Enfim, coisas simples que levantam o ânimo de estudar e por a mão na massa e não ficar só tickando checklists :)

**Comment by evelew** [comment]
O que me ajuda bastante é mexer com coisas diferentes, por exemplo, eu trabalho como frontend então tô sempre vendo coisas relacionadas a web, quando sinto que tô ficando desanimada tento fazer algo em outra área, automatizar alguma tarefa, criar alguma extensão do editor que eu uso e etc.

E sobre saber não saber mais o que aprender, quando me senti assim comecei a seguir pessoas que tem **mais** experiência que eu, isso tb tem me ajudado bastante.

**Comment by lucianobarauna** [comment]
To nesse mesmo barco @manzolirch . Fiz e faço muito com css mais o JS ainda é um pouco devagar.

Essa thread veio numa hora boa. Passei por esse lapso tem alguns dias e acho que fiz um pouco de cada coisa que @mstuttgart falou.

O CSS eu sempre procuro me manter atualizado é mais fácil de por conta do meu conhecimento e lido toda hora com isso no trabalho mas queria me atua
lizar profissionalmente e me divertir ao mesmo tempo. Infelizmente o meu trampo atual não dá para sair implementando a onda do momento de frameworks JS e por conta disso você fica meio preocupado em ficar para trás em algum ponto.

Por conta disso faz alguns dias eu estava dando uma surtada de leve, o que eu fiz ?

Falei com amigos mais experientes, tentei me expressar da melhor forma e pedi opiniões. Isso me ajudou muito e hoje diminui a pressão de cobrança que estava em cima de mim e voltei a estudar :-)

Dependendo da pessoa uma boa conversa é sempre bem vinda. Além das dicas que @mstuttgart passou que foram mais ou menos o que fiz sem ter chego nesse post primeiro rs.

**Comment by wilsonneto-swe** [comment]
Eu me mantenho animado em sempre aprender fazendo projetos pessoais com tecnologias que gosto ou que gosto de aprender... Acho a melhor coisa, gosto muito do trabalho, aprendo e implemento muita coisa legal, mas la tem a pressão, o prazo... Em meus projetos pessoais é a hora em que desenvolvimento é apenas por diversão, atualmente estou estudando React Native e Flutter juntos, muitos me falar para estudar um e depois o outro, mas cara é assim que eu me divirto, crio uma tela no Flutter ai meu desafio é criar a mesma em RN... Nesta pegada de projetos pessoais por diversão fiz um app de guia de eventos para minha cidade, que morreu já rs Um ecommerce de venda de ingressos que até hoje me gera uns trocados, e agora estou fazendo um novo app para brincar com o ReactMUI no front, Flutter e RN nos apps. E para ser sincero me divirto com isso, e daqui a pouco brincando vou estar bem afiado em ambas as tecnologias.

Espero ter ajudado, abraços!

**Comment by jpferreiradev** [comment]
Boa noite pessoal, 

Eu estou passando por isso, principalmente com a motivação para o  estudo, eu sei que eu amo essa área e sei do que ela pode nos trazer de mal, recentemente acabei mudando de área, era de infra/suporte e agora eu estou com desenvolvimento, e até conseguir um estágio que durou menos que de um ano, e sei o quanto é importante estudar, mas para mim sempre foi uma coisa complicada, tudo o que eu aprende com a TI foi sozinho, mal sentava para estudar realmente, e até nos empregos, aprendia tudo na prática e  desenvolvimento precisa de uma rotina de estudos, por diversos motivos, mas isso acabou me deixando meio mal, e agora estou em busca de ter uma disciplina para estudar, vocês já passaram por isso?

**Comment by ninetails** [comment]
@jotepauder parabéns pela mudança de área man!

Bom, eu não irei responder em si a thread pq eu acho que também estou do lado dos que estão perdidos.

Mas disso eu sempre lembro duma lição que tive dum professor no cursinho falando sobre _hobbies_ (quotando mas não são as palavras exatas que ele usou pq eu mal lembro):

> Gosto de marcenaria. Não trabalho com isso pois gosto de deixar como _Hobby_. Talvez se eu trabalhar com isso e ter que levar a sério marcenaria, talvez eu deixe de gostar de marcenaria. Hoje sou professor. Trabalho como professor não porquê eu gosto 100% disso. Trabalho para poder sustentar meu hobby.

Há quem goste do que faz, há quem não goste. Existem aqueles que são apaixonados pelo que faz, existem outros que nem tanto. Sempre gostei de jogos, mas durante minha vida nunca me vi apaixonado por ser gamedev. O que sempre gostei, mas não fui apaixonado foi por web.

O que me dificulta em meio ao regime de trabalho remoto por causa da pandemia é que, por não ter um lugar próprio para trabalhar como um escritório em casa dedicado, eu acabo perdendo o foco em relação ao trabalho.

Mas nesses últimos dias andei filosofando sobre isso e creio que isso meio que vale pra vida em si. Tudo bem que quem te emprega espera que você trabalhe 100% para ele no tempo que vc estiver em trabalho. Mas mesmo nosso trabalho quando fazíamos no escritório não era 100% dedicado às entregas, havia momentos de reunião, haviam momentos de reunião informais, cafés com a galera... tempo pra discutir sobre o trabalho e sobre o ambiente.

Mesmo eu me cobrando por estar vendo menos sobre assuntos web, vi que não é bem assim. Cada um tem um passo e seu tempo. Cada um tem seu _hobby_, algo pra refrescar a cabeça Precisamos desse tempo. E acho que com isso e vc tendo a mentalidade de que você faria melhor de outro jeito, vc começa a se forçar no seu ritmo a ter que dar mais importância às coisas que vc deixou de lado, mesmo pq essa cobrança na maioria das vezes não vai vir de cima.

Acho que se a autocobrança nos fala que estamos deixando de fazer algo de produtivo, e ela quase sempre está certa... Porém temos tanto quanto fazer esse algo produtivo também temos que fazer algo que nos suporte nos deixando animados a certo ponto de suportar fazer esse algo produtivo. Ou caso for estudos sem _deadline_, pause o atual e mude para algo que te instiga um maior interesse. Depois vc volta nele.

Sendo didático, é bem aqueles _trade-off_ de pais, mas de nós para nós mesmos:
> Você quer jogar? Pode jogar, mas só se depois de jogar, arrumar seu quarto.

E também não acho saudável ficar comparando "ah mas na época da faculdade eu estudava e trabalhava, hj n tenho o mesmo ritmo". Cada época é uma época. Da mesma forma que eu não acho que o rendimento meu agora é o mesmo daquela época, não será caso eu (sei lá, dando um exemplo mais extremo) tenha filhos.

E no final tentar adaptar e achar um ritmo aceitável. Querendo ou não, vc tem que estar bem pra poder trabalhar, bem para poder focar, bem pra poder estudar. Se vc não tá bem: diminua o ritmo ou uma alternativa seria procurar algum tratamento (não estou falando aqui pejorativamente, às vezes a gente entra numas que vemos que realmente precisamos de um psicólogo).

Sei lá, só estou divagando e desabafando...

**Comment by jpferreiradev** [comment]
@ninetails Eu que agradeço man! Obrigado!

Primeiro obrigado pelo texto, existem coisas que nós mesmos fazemos que acabamos nos desmotivando, e nem percebemos,  seja se comparando, seja se exigindo demais nas nossas 
próprias cobranças, e principalmente não cuidando da nossa saúde mental, eu vejo isso como a base para nós termos uma vida mais tranquila, e principalmente se conhecendo, eu como você também amo jogos, até pensei em aprender a desenvolver jogos e trabalhar com isso, mas é totalmente diferente, kkkkkk! Eu sei com o que eu quero trabalhar em desenvolvimento, mas sei também como essa área "exige" que você tenha um estudo constante, para você cada vez mais  melhorar, e é isso que eu preciso melhorar em mim. E concordo também uma coisa que você falou no final que você precisa estar bem para fazer as suas coisas, e eu fazia terapia e agora vou voltar.

Se cuida cara, abraço!
